backend/src/main/java/com/zrcaw/langshub/controller/translate/TranslateController.java

```diff
@@ -0,0 +1,39 @@
+package com.zrcaw.langshub.controller.translate;
+
+import com.zrcaw.langshub.dto.translate.TranslateRequest;
+import com.zrcaw.langshub.dto.translate.TranslateResponse;
+import com.zrcaw.langshub.service.translate.TranslateService;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@RestController
+@RequestMapping("/translate")
+public class TranslateController {
+
+    private final TranslateService translateService;
+
+    public TranslateController(TranslateService translateService) {
+        this.translateService = translateService;
+    }
+
+    @PostMapping("/text")
+    public ResponseEntity<TranslateResponse> translateText(@RequestBody TranslateRequest request) {
+        return ResponseEntity.ok(translateService.translateText(request));
```

Comment:
Według mnie języki powinniśmy już tutaj weryfikować, czy ktoś dobrze wpisał, a nie pozwalać się wykrzaczyć awsowi, jeśli bedzie tylko ang-pl i pl-ang to można to jakoś uogólnić do dwóch enumów.